.circleci/windows/test.bat

```diff
@@ -0,0 +1,66 @@
+@echo on
```

Comment:
This script simulates user behavior. It extracts the .zip artifact and runs smoke tests